### DISCUSSION TITLE
What are the differences from CVA?  And why use this and not CVA?

### DISCUSSION THREAD

**Discussion by its-monotype** [heroui-inc/tailwind-variants#13] [General]
As title says

**Comment by jrgarciadev** [comment]
Hey @its-monotype 👋🏻

Please check this comment https://github.com/nextui-org/nextui/discussions/1035#discussioncomment-4876531

We will add a comparison page to the docs, you could also compare the source code to see all the differences 👍🏻

**Comment by its-monotype** [comment]
@jrgarciadev I'm excited that now the ability to create UI kits is much easier because I like stitches approach and it's amazing that we can use it with TailwindCSS
